src/spatio_temporal_voxel_layer.cpp

```diff
@@ -902,6 +902,33 @@ SpatioTemporalVoxelLayer::dynamicParametersCallback(std::vector<rclcpp::Paramete
         }
       }
     }
+
+    if (type == ParameterType::PARAMETER_BOOL) {
+      if (name == name_ + "." + "enabled") {
+        bool enable = parameter.as_bool();
+        if (enabled_ != enable) {
+          if (enable){
+            observation_subscribers_iter sub_it = _observation_subscribers.begin();
+            for (; sub_it != _observation_subscribers.end(); ++sub_it) {
+              (*sub_it)->subscribe();
+            }
+
+            observation_buffers_iter buf_it = _observation_buffers.begin();
+            for (; buf_it != _observation_buffers.end(); ++buf_it) {
+              (*buf_it)->ResetLastUpdatedTime();
+            }
+          } else {
+            observation_subscribers_iter sub_it = _observation_subscribers.begin();
+            for (; sub_it != _observation_subscribers.end(); ++sub_it) {
+              if (*sub_it != NULL) {
```

Comment:
Why do you check here if any are nullptrs but not above?

Reply:
It's because I copied and pasted...
Do you think the nullptr inspection part is necessary?
I don't think that part is necessary.
If you agree, should I also remove it from the "deactivate" function?

Reply:
Add it here